doc/modules/language-guide/examples/grammar.txt

```diff
@@ -299,6 +299,7 @@
 
 <imp> ::= 
     'import' <id>? '='? <text>
```

Comment:
I never undestood why we allo anonymous imports.... especially if they have to be pure...

Reply:
I believe anonymous imports are a leftover from when imports still were expressions.

Reply:
these are gone in 8d7fb3cae!